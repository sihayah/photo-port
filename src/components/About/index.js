import React from 'react';
import coverImage from '../../assets/cover/cover-image.jpg';

function About() {
    return (
        <section className="my-5">
            <h1 id="about">
                Who am I?
            </h1>
            <img src={coverImage} className="my-2" style={{ width: "100% "}} alt="cover"/>
            <p>
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Cumque, aut nam aperiam voluptatem tempore fugiat delectus sunt? Numquam eos fugit quo asperiores libero tempore excepturi expedita, perspiciatis magnam, sit ab.
            </p>
        </section>
    )
}

export default About;