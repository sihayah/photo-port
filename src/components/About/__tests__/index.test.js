import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import About from '..';

afterEach(cleanup);

describe('About component', () => {
    // 1st test
    it('renders', () => {
        render(<About />);
    });
    // 2nd test
    it('match snapshot DOM node structure', () => {
        // render About
        const { asFragment } = render(<About />);
        expect(asFragment()).toMatchSnapshot();
    })

    
})