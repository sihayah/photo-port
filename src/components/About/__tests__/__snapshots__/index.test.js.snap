// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component match snapshot DOM node structure 1`] = `
<DocumentFragment>
  <section
    class="my-5"
  >
    <h1
      id="about"
    >
      Who am I?
    </h1>
    <img
      alt="cover"
      class="my-2"
      src="cover-image.jpg"
      style="width: 100%;"
    />
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Cumque, aut nam aperiam voluptatem tempore fugiat delectus sunt? Numquam eos fugit quo asperiores libero tempore excepturi expedita, perspiciatis magnam, sit ab.
    </p>
  </section>
</DocumentFragment>
`;
